pipelines/optimus/test_optimus_full_datasets/ischaemia_5/submit.sh

```diff
@@ -0,0 +1,12 @@
+skylab_path="/Users/nbarkas/work/optimus/runs_Mar19/skylab"
```

Comment:
Remove these skylab_path variable values and either make them relative or make them so that the user must specify manually

Reply:
@barkasn Is this just for the ischaemia_5 dataset or all example datasets? What would the relative path be?